charts/presidio/templates/recognizers-store-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+{{ $fullname := include "presidio.recognizersstore.fullname" . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $fullname }}
+  labels:
+    app: {{ $fullname }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ $fullname }}
+  template:
+    metadata:
+      labels:
+        app: {{ $fullname }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.registry }}/{{ .Values.recognizersstore.name }}:{{ default .Chart.AppVersion .Values.tag }}"
+        imagePullPolicy: {{ default "IfNotPresent" .Values.recognizersstore.imagePullPolicy }}
+        ports:
+        - containerPort: {{ .Values.recognizersstore.service.internalPort }}
+        resources:
+          requests:
+            memory: "1500Mi"
```

Comment:
This seems like a pretty large memory allocation for a go service or am I missunderstanding.

Reply:
Correct. typo